<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XPlayer</class>
 <widget class="QWidget" name="XPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>XPlayer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/Resources/logo.ico</normaloff>:/Resources/logo.ico</iconset>
  </property>
  <widget class="VideoWidget" name="openGLWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
  </widget>
  <widget class="XSlider" name="playSlider">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>470</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="playTime">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 20pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>000:00</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openBtn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>520</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:!hover{border-image: url(:/Resources/open_normal.png);}
QPushButton:hover{border-image: url(:/Resources/open_hot.png);}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="playBtn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>520</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:!hover{border-image: url(:/Resources/play_normal.png);}
QPushButton:hover{border-image: url(:/Resources/play_hot.png);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="sp">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>520</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 20pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalTime">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>520</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 20pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>000:00</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>videowidget.h</header>
  </customwidget>
  <customwidget>
   <class>XSlider</class>
   <extends>QSlider</extends>
   <header>xslider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>openBtn</sender>
   <signal>clicked()</signal>
   <receiver>XPlayer</receiver>
   <slot>open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>539</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>625</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playSlider</sender>
   <signal>sliderPressed()</signal>
   <receiver>XPlayer</receiver>
   <slot>sliderPress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>476</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>748</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>XPlayer</receiver>
   <slot>sliderRelease()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>479</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>739</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playBtn</sender>
   <signal>clicked()</signal>
   <receiver>XPlayer</receiver>
   <slot>play()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>541</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>606</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>open()</slot>
  <slot>sliderPress()</slot>
  <slot>sliderRelease()</slot>
  <slot>play()</slot>
 </slots>
</ui>
